electricitymap/contrib/lib/models/event_lists.py

```diff
@@ -69,8 +76,50 @@ def append(
         if event:
             self.events.append(event)
 
+    @staticmethod
+    def merge_production_breakdowns(
+        ungrouped_production_breakdowns: List["ProductionBreakdownList"],
+        logger: Logger,
+        merge_zone_key: ZoneKey,
```

Comment:
The zone key should also always be the same between the imputs to prevent accidental merges between diffrent zones. Maybe we can just compare thes and set it in the method instead of defining it here?

Reply:
Agree there should be at least some safe guards